### PR TITLE
71 Created directive to show and hide password

### PR DESCRIPTION
**Pull request by ShevchenkoKirilo**
image
This directive can be applied to any password field

### CODE REVIEW

**Comment by ShevchenkoKirilo** [comment]
image
image

**Comment by Anatolii-Ivanov** [comment]
Make right padding the same as left

**Comment by ShevchenkoKirilo** [comment]
> Make right padding the same as left

image

**Review by Anatolii-Ivanov — APPROVED** [review]

File: frontend/src/app/shared/directives/password-visibility.directive.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Directive, ElementRef } from '@angular/core';
+
+@Directive({
+    selector: '[appPasswordVisibility]',
+})
+export class PasswordVisibilityDirective {
```
Comment (tatianahutii) [inline frontend/src/app/shared/directives/password-visibility.directive.ts:6]: You can implement this functionality without directive

Comment (tatianahutii) [reply frontend/src/app/shared/directives/password-visibility.directive.ts:6]: https://nitishkaushik.com/show-hide-password-in-angular-with-angular-material/

**Review by tatianahutii — CHANGES_REQUESTED** [review]

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**